cli/src/index.js

```diff
@@ -182,7 +183,32 @@ function progressTracker(results) {
   return tracker;
 }
 
+async function checkInputFilesValid(files) {
+  const validFiles = [];
+
+  for (const file of files) {
+    try {
+      await fsp.stat(file);
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        console.warn(
+          `Warning: Input file does not exist: ${resolvePath(file)}`,
+        );
+        continue;
+      } else {
+        throw err;
+      }
+    }
+
+    validFiles.push(file);
+  }
+
+  return validFiles;
+}
+
 async function processFiles(files) {
+  files = await checkInputFilesValid(files);
```

Comment:
I'm not sure how people feel about mutating an argument like this. I can refactor if necessary.

Reply:
I think that’s absolutely fine!